sandbox/apparmor/apparmor.go

```diff
@@ -673,6 +673,15 @@ func probeParserFeatures() ([]string, error) {
 			probe:   "mqueue,",
 			minVer:  "4.0.1",
 		},
+		{
+			feature: "allow-all",
+			probe:   "allow all,",
+			minVer:  "4.0.2",
+		},
+		{
+			feature: "io-uring",
+			probe:   "allow io_uring,",
```

Comment:
I wonder if this should be just `io_uring,` (ie drop the `allow` prefix since we aren't using that on any other probes...? Or should we update the other probes to include an explicit `allow` as well?

Reply:
I think I had copy-pasted from allow all, I can drop the prefix.